src/Service.Tests/Unittests/ConfigValidationUnitTests.cs

```diff
@@ -1982,17 +1971,23 @@ string[] linkingTargetFields
                 Entities: new(entityMap)
             );
 
+            MockFileSystem fileSystem = new();
```

Comment:
This block of code is common among few tests. Can we scope it out into another helper which will return the instance of RuntimeConfigValidator that we need?

Reply:
Thanks for the suggestion. Introduced a helper method.